pegasus/helpers/page_helpers.rb

```diff
@@ -29,3 +29,7 @@ def partner_site?()
   partner_sites = CDO.partners.map{|x|x + '.code.org'}
   return partner_sites.include?(request.site)
 end
+
+def mobile?
+  return !(request.user_agent =~ /Mobile/).nil?
```

Comment:
we have a very different check for mobile devices in learn-carousels.haml.  which one do we trust more?